src/components/Frame/Frame.scss

```diff
@@ -1,6 +1,6 @@
 $button-size: rem(32px);
-$transition-duration: 260ms;
```

Comment:
This PR adds `40ms` to this transition, as `260ms` is going up to `300ms` (`duration-slow = 300ms`), was there agreement reached to do this? Also: should we mention this change in the release notes?

Reply:
To answer to @beefchimi I think this was an oversight, probably just ported over when we moved frame from web. I'll 🎩